### PR TITLE
Flow

### PR DESCRIPTION
**Pull request by dariusf**
Let's try Flow? :bowtie: :sparkles: 

Depends on #6.

### CODE REVIEW

**Comment by LowWeiLin** [comment]
I'm trying to run it but having troubles on windows.

**Comment by LowWeiLin** [comment]
1) Windows needs this binary https://github.com/facebook/flow/releases/download/v0.33.0/flow-win64-v0.33.0.zip

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "mineflayer-blockfinder": "0.0.5",
+    "mineflayer-navigate": "0.0.9"
+  },
+  "devDependencies": {
+    "flow-bin": "^0.33.0",
+    "flow-remove-types": "^1.0.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "flow; test $? -eq 0 -o $? -eq 2; flow-node"
```
Comment (LowWeiLin) [inline package.json]: This looks like shell code? I can't run this on windows. =(

Comment (LowWeiLin) [reply package.json]: The windows example uses a different script here: https://flowtype.org/blog/2016/08/01/Windows-Support.html

"flow_check": "flow check || exit 0"

Not sure if it does the same thing?

Comment (dariusf) [reply package.json]: Oops, the `test` is redundant. The `exit 0` makes it so that if type-checking fails, `npm` won't print its error messages.

**Comment by dariusf** [comment]
Added a library definition for mineflayer. I was lazy and marked some as `any` or `Function`, but the more specific ones work well. You may need to `flow check` to pick it up.

@LowWeiLin could you help to check/update the Windows script?

**Comment by LowWeiLin** [comment]
Fixed the windows script

**Pull request merged**